### DISCUSSION TITLE
整合包出问题里，各位帮忙看看日志呗

### DISCUSSION THREAD

**Discussion by xingnengmao666** [Meloong-Git/PCL#1603] [提问] [answered]
[178370299.zip](https://github.com/Hex-Dragon/PCL2/files/11182853/178370299.zip)

**Comment by LTCatt** [comment, marked as answer]
游戏崩溃一般和启动器无关，你可以问问其他人……

**Comment by GTEMYee** [comment]
会不会是Honor Virtual Display Device
crash-cilent.txt内显示它在Graphics card #0
